EnglishFilter/sections/antiadblock.txt

```diff
@@ -708,6 +708,9 @@ smallseotools.com#@#.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/122627
 ||read.amazon.*/ads-prebid.js$script,redirect=prebid-ads,important
 @@||read.amazon.*/ads-prebid.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/122663
+kentonline.co.uk#$##nagBG { display: none !important; }
+kentonline.co.uk#$#body { overflow: auto !important; }
```

Comment:
@Alex-302 @jellizaveta what about AdGuard Content Blocker? These rules does not work there.

Reply:
When CB user will report this issue, we will check it again)

Reply:
Previous anti-adblock did not lead to rerirect to a dummy page...